store/postgres/src/relational.rs

```diff
@@ -652,17 +722,23 @@ impl Table {
             block_range = BLOCK_RANGE
         )?;
 
-        for (i, column) in self.columns.iter().enumerate() {
+        // Create indexes. Don't bother with enum types, since an index on a
+        // column that can take only a handful of values is pointless
```

Comment:
This may be a controversial topic, but as we discussed when we considered not indexing boolean columns, I don't think it's pointless to index a column of low cardinality. It is pointless if the values are uniformly distributed, but if some values occur rarely than an index can be useful. To me, "index everything" still seems like a good strategy.

Reply:
I don't want to hold this PR up with this debate, so changed the code to index enum columns. But I would like to see a query plan from production where an index on a boolean column actually gets used, or at the very least some evidence from `pg_stat_user_indexes` that some boolean index has been used and we are not just slowing down writes unnecessarily with these indexes.